packages/std/src/results/cosmos_msg.rs

```diff
@@ -160,6 +163,23 @@ pub enum WasmMsg {
     ClearAdmin { contract_addr: String },
 }
 
+#[cfg(feature = "stargate")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum GovMsg {
+    Vote { proposal_id: u64, vote: Vote },
+}
+
+#[cfg(feature = "stargate")]
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum Vote {
+    Yes,
+    No,
+    Abstain,
+    Veto,
```

Comment:
This should be

```suggestion
    NoWithVeto,
```

See https://github.com/cosmos/cosmos-sdk/blob/v0.42.5/proto/cosmos/gov/v1beta1/gov.proto#L17-L30